amplitude/logger.go

```diff
@@ -6,34 +6,34 @@ import (
 )
 
 type Logger interface {
-	Debug(message string, args ...interface{})
-	Info(message string, args ...interface{})
-	Warn(message string, args ...interface{})
-	Error(message string, args ...interface{})
+	Debugf(message string, args ...interface{})
+	Infof(message string, args ...interface{})
+	Warnf(message string, args ...interface{})
+	Errorf(message string, args ...interface{})
 }
 
 type stdLogger struct {
 	logger *log.Logger
 }
 
-func (l *stdLogger) Debug(message string, args ...interface{}) {
+func (l *stdLogger) Debugf(message string, args ...interface{}) {
 	l.logger.Printf("Debug: "+message, args...)
 }
 
-func (l *stdLogger) Info(message string, args ...interface{}) {
+func (l *stdLogger) Infof(message string, args ...interface{}) {
 	l.logger.Printf("Info: "+message, args...)
 }
 
-func (l *stdLogger) Warn(message string, args ...interface{}) {
+func (l *stdLogger) Warnf(message string, args ...interface{}) {
 	l.logger.Printf("Warn: "+message, args...)
 }
 
-func (l *stdLogger) Error(message string, args ...interface{}) {
+func (l *stdLogger) Errorf(message string, args ...interface{}) {
 	l.logger.Printf("Error: "+message, args...)
 }
 
-func newDefaultLogger() Logger {
-	return &stdLogger{logger: log.New(os.Stderr, "amplitude-analytics", log.LstdFlags)}
+func NewDefaultLogger() Logger {
+	return &stdLogger{logger: log.New(os.Stderr, "amplitude-analytics ", log.LstdFlags)}
```

Comment:
Minor, probably add `:` or `- ` too? `amplitude-analytics: `, `amplitude-analytics - `

Reply:
It's more visually friendly. Added `-` here, as `:` is used else where